### PR TITLE
update installation scripts

### PR DESCRIPTION
**Pull request by memeramita**
<h2>Context</h2>

Since we changed the python version from 3.9 to 3.10 and 3.11, the installation scripts need to be updated. 

<h2>Changes</h2>

I changed the python version in the conda command to 3.10 and updated the comment in both `install_napari.command` and `install_napari.sh`.

### CODE REVIEW

File: napari_allencell_annotator/assets/install_napari.command [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
```
Comment (yrkim98) [inline napari_allencell_annotator/assets/install_napari.command]: I thought we were using zsh for this?

Comment (memeramita) [reply napari_allencell_annotator/assets/install_napari.command]: fixed in https://github.com/aics-int/napari-allencell-annotator/pull/97/commits/83bbf28390b071bb9d89bd14db8357f40ae05a7d

**Comment by yrkim98** [comment]
dont merge this until it works for you

**Comment by yrkim98** [comment]
wont merge, getting rid of install scripts.

**Pull request closed without merging**